app/graphql/types/address_type.rb

```diff
@@ -0,0 +1,15 @@
+module Types
+  class AddressType < Types::BaseObject
+
+    description 'A partner'
+
+    # field :id, ID, null: false
+    field :street_address, String
+    field :street_address2, String
+    field :street_address3, String
```

Comment:
could we also give people access to `address.full_street_address` and `address.all_address_lines` from the model just for convenience?